api_v3/lib/types/KalturaUserEntryStatus.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/**
+ * @package api
+ * @subpackage enum
+ */
+class KalturaUserEntryStatus extends KalturaEnum
```

Comment:
Please implement UserEntryStatus and move the constants to UserEntryStatus.
Also, this enum should be dynamic.